tests/functional/test_functional.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import unittest
+import requests_ntlm
+
+username = '.\\User'
+password = 'Password01'
+http_with_cbt = 'http://127.0.0.1:81/contents.txt'
+http_without_cbt = 'http://127.0.0.1:82/contents.txt'
+https_with_cbt = 'https://127.0.0.1:441/contents.txt'
+https_without_cbt = 'https://127.0.0.1:442/contents.txt'
+expected = 'contents'
+
+class Test_Functional(unittest.TestCase):
```

Comment:
It'll run fine. You're running the tests via py.test, which doesn't require that the test classes inherit from `unittest.TestCase`.